packages/server/src/migrations/schema/v69.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Generated by @medplum/generator
+ * Do not edit manually.
+ */
+
+import { PoolClient } from 'pg';
+
+export async function run(client: PoolClient): Promise<void> {
+  await client.query('ALTER TABLE IF EXISTS "Subscription" ADD COLUMN IF NOT EXISTS "author" TEXT');
```

Comment:
Hm, this doesn't seem right... I think this should have used the `Subscription_References` table, not added a new column 🤔 

This may be a bug in the migration generator, due to our custom `Meta` types 🤔

Reply:
I don't think the migration generator has been updated to remove the `Reference` columns yet; it's probably good to keep both until we fully deprecate the old columns altogether